bazel/external/wee8.genrule_cmd

```diff
@@ -109,6 +109,7 @@ if [[ $${ARCH} == "x86_64" ]]; then
   WEE8_BUILD_ARGS+=" target_cpu=\"x64\""
 elif [[ $${ARCH} == "aarch64" || $${ARCH} == "arm64" ]]; then
   WEE8_BUILD_ARGS+=" target_cpu=\"arm64\""
+  export LDFLAGS="$${LDFLAGS-} -z max-page-size=0x10000"
```

Comment:
Without this line, v8 compilation gets a segfault. The commit message explains the root cause.

Reply:
Can you add a comment to the end of this line reminding us to remove this line after we upgrade?